### PR TITLE
Feature/MDS-2178 Hide fields that will be overridden by the ETL when editing a permit amendment for a regional mine.

### PR DESCRIPTION
**Pull request by nayr974**
After talking with Aaron he only intends for Admins to upload documents at this time, so no additional roles or authorization changes. As we are still ETLing and overwriting regional mine permit information, I have hidden fields other than documents on the edit form just in case an Admin assumes they can modify these fields while uploading documents.

### CODE REVIEW

**Comment by pdittaro** [comment]
I don't think that this PR description matches the changes contained. Why are you rendering the select party and description components on the PermitAmendmentForm for major mines?

File: frontend/src/tests/components/Forms/__snapshots__/PermitAmendmentForm.spec.js.snap [hunk 1]
```diff
@@ -10,58 +10,6 @@ exports[`PermitAmendmentForm renders properly 1`] = `
...
   >
-    <Col
-      md={12}
-      sm={24}
-    >
-      <FormItem
-        hasFeedback={false}
-      >
-        <Connect(PartySelectField)
-          allowAddingParties={true}
```
Comment (Jsyro) [inline frontend/src/tests/components/Forms/__snapshots__/PermitAmendmentForm.spec.js.snap]: Just pointing out that the snapshot test does not render the component with isMajorMine is true. Is that intended.

**Comment by nayr974** [comment]
> 
> 
> I don't think that this PR description matches the changes contained - are you able to update what these changes encompass?

Renamed the title to better describe the outcome of the task.

**Comment by pdittaro** [comment]
Very succinct title, thank you.

File: frontend/src/tests/components/Forms/__snapshots__/PermitAmendmentForm.spec.js.snap (hunk 1 above)
Comment (nayr974) [reply frontend/src/tests/components/Forms/__snapshots__/PermitAmendmentForm.spec.js.snap]: set isMajorMine=true in the test which now renders the full component

**Pull request merged**